Package.swift

```diff
@@ -17,10 +17,10 @@ let libwordpressFFI: Target = libwordpressFFIVersion.target
 var package = Package(
     name: "WordPressAPI",
     platforms: [
-        .iOS(.v15),
+        .iOS(.v16),
```

Comment:
I'm not a big fan of bumping the minimum Swift version just for tests, but iOS 15 doesn't support Swift Regex literals. The fact that Swift features aren't back ported that far makes me think nobody needs that.

This would be easy enough to walk back, so I don't mind making the change and we can re-evaluate later.